app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx

```diff
@@ -61,15 +57,13 @@ export class PatternControlPanel extends Component {
     readOnly: false,
   };
 
-  onEditPattern = () => {
-    const { intl, showModal, updatePattern, pattern } = this.props;
+  onRenamePattern = () => {
```

Comment:
Seems that you forget to rename function [here](https://github.com/reportportal/service-ui/pull/1597/files#diff-8da86751ed05967963f336b5325068a7R134)

Reply:
Fixed.